public/src/main/java/com/revenuecat/purchases/parceler/JSONObjectParceler.kt

```diff
@@ -5,7 +5,7 @@ import kotlinx.parcelize.Parceler
 import org.json.JSONObject
 
 /** @suppress */
-internal object JSONObjectParceler : Parceler<JSONObject> {
+object JSONObjectParceler : Parceler<JSONObject> {
```

Comment:
Hmm should we move this somewhere else? Maybe the `utils` module? Otherwise we would be exposing this class to users I believe which I don't think we want.